CS3230Project/View/EditPatient.cs

```diff
@@ -0,0 +1,67 @@
+﻿using CS3230Project.Model.Accounts;
```

Comment:
Not holding up the approval. The edit patient should be in a separate branch as they are separate features and therefore a separate PR, unless I am missing anything. I know the edit patient work is still in progress so I'll approve it although it does need some work.

Reply:
That was an accident on my part.